tailwind.config.js

```diff
@@ -47,6 +49,13 @@ module.exports = {
           'monospace',
         ],
       },
+      padding: {
+        yt: '56.25%',
+      },
+      flex: {
+        1: '1',
```

Comment:
where is this used?

Reply:
In the HowItWorks sections , to make the image and the text having equal space.

Also used in the youtube embed section and create an account section.

Reply:
flex-1 is already defined within Tailwind default config: https://tailwindcss.com/docs/flex
You can remove '1' and add '2' similarly if we are using.